packages/fluentui/react-northstar/src/components/Tree/hooks/useTree.ts

```diff
@@ -236,6 +243,43 @@ export function useTree(options: UseTreeOptions): UseTreeResult {
     [getItemById, getItemRef],
   );
 
+  const searchByFirstChar = React.useCallback(
+    (startIndex: number, endIndex: number, char: string) => {
+      for (let i = startIndex; i < endIndex; ++i) {
+        // get first charater of tree node using the same way aria does (https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-2/js/treeitemLinks.js)
+        const itemFirstChar = getItemRef(visibleItemIds[i])
+          ?.textContent?.trim()
+          ?.substring(0, 1)
+          ?.toLowerCase();
+        if (itemFirstChar === char) {
+          return i;
+        }
+      }
+      return -1;
+    },
+    [getItemRef, visibleItemIds],
+  );
+
+  const setToFocusIDByFirstCharacter = React.useCallback(
+    (e: React.KeyboardEvent, idToStartSearch: string) => {
+      // Get start index for search
+      let starIndex = visibleItemIds.indexOf(idToStartSearch) + 1;
+      if (starIndex === visibleItemIds.length) {
+        starIndex = 0;
+      }
+
+      // Check following nodes in tree
+      let toFocusIndex = searchByFirstChar(starIndex, visibleItemIds.length, e.key);
+      // If not found in following nodes, check from beginning
+      if (toFocusIndex === -1) {
+        toFocusIndex = searchByFirstChar(0, starIndex - 1, e.key);
```

Comment:
Is it intentional that upper case characters are not supported ?

Reply:
The letter key navigation is case insensitive. because 'a' and 'A' have the same keycode

Reply:
Oh I probably miss understood you. This is a mistake on my side. I changed to use `e.key.toLowerCase()`